docs/_docs/reference/changed-features/changed-features.md

```diff
@@ -2,6 +2,7 @@
 layout: index
 title: "Other Changed Features"
 movedTo: https://docs.scala-lang.org/scala3/reference/changed-features.html
+redirectFrom: /changed-features.html
```

Comment:
Actually, I just realized that this is going to create an unexpected URL in the Dotty website (it should be `/reference/changed-features.html` in the Dotty website). Do you see a possible solution, @pikinier20?

Reply:
In e17894ef4a I propose an alternative implementation that alters only the content we publish to https://docs.scala-lang.org/scala3/reference.